### PR TITLE
Move html code to templates and use printf/sprintf functions

### PR DESCRIPTION
**Pull request by vishalkakadiya**

### CODE REVIEW

File: app/main/controllers/template/RTMediaNav.php [hunk 1]
```diff
@@ -281,7 +281,18 @@ public function sub_nav() {
...
 			$all = 'current selected';
 		}
-		echo apply_filters( 'rtmedia_sub_nav_all', '<li id="rtmedia-nav-item-all-li" class="' . esc_attr( $all ) . '"><a id="rtmedia-nav-item-all" href="' . esc_url( trailingslashit( $link ) ) . RTMEDIA_MEDIA_SLUG . '/">' . esc_html__( 'All', 'buddypress-media' ) . '<span class="count">' . esc_html( ( isset( $counts['total']['all'] ) ) ? rtmedia_number_to_human_readable( $counts['total']['all'] ) : 0 ) . '</span>' . '</a></li>' );// @codingStandardsIgnoreLine
+
+		$html_li = sprintf(
+			'<li id="rtmedia-nav-item-all-li" class="%1$s">
+				<a id="rtmedia-nav-item-all" href="%2$s">%3$s<span class="count">%4$s</span></a>
+			</li>',
+			esc_attr( $all ),
+			esc_url( trailingslashit( $link ) ) . RTMEDIA_MEDIA_SLUG . '/',
```
Comment (chandrapatel) [inline app/main/controllers/template/RTMediaNav.php:290]: Escape full URL.

File: app/main/controllers/template/RTMediaNav.php [hunk 2]
```diff
@@ -302,7 +313,18 @@ public function sub_nav() {
...
 			$counts['total']['album'] = $counts['total']['album'] + $other_count;
 			$album_label              = esc_html__( defined( 'RTMEDIA_ALBUM_PLURAL_LABEL' ) ? constant( 'RTMEDIA_ALBUM_PLURAL_LABEL' ) : 'Albums', 'buddypress-media' );
-			echo apply_filters( 'rtmedia_sub_nav_albums', '<li id="rtmedia-nav-item-albums-li" class="' . esc_attr( $albums ) . '"><a id="rtmedia-nav-item-albums" href="' . esc_url( trailingslashit( $link ) ) . RTMEDIA_MEDIA_SLUG . '/album/">' . esc_html( $album_label ) . '<span class="count">' . esc_html( ( isset( $counts['total']['album'] ) ) ? rtmedia_number_to_human_readable( $counts['total']['album'] ) : 0 ) . '</span>' . '</a></li>' );// @codingStandardsIgnoreLine
+
+			$html_li = sprintf(
+				'<li id="rtmedia-nav-item-albums-li" class="%1$s">
+				<a id="rtmedia-nav-item-albums" href="%2$s">%3$s<span class="count">%4$s</span></a>
+			</li>',
+				esc_attr( $albums ),
+				esc_url( trailingslashit( $link ) ) . RTMEDIA_MEDIA_SLUG . '/album/',
```
Comment (chandrapatel) [inline app/main/controllers/template/RTMediaNav.php:322]: Escape full URL.

File: app/main/controllers/template/RTMediaNav.php [hunk 3]
```diff
@@ -302,7 +313,18 @@ public function sub_nav() {
...
+				esc_html( $album_label ),
+				esc_html( ( isset( $counts['total']['album'] ) ) ? rtmedia_number_to_human_readable( $counts['total']['album'] ) : 0 )
+			);
+
+			echo apply_filters( 'rtmedia_sub_nav_albums', $html_li );// @codingStandardsIgnoreLine
```
Comment (chandrapatel) [inline app/main/controllers/template/RTMediaNav.php:327]: This syntax of ignoring PHPCS is deprecated. Use `// phpcs:ignore <rule>`.

File: app/main/controllers/template/RTMediaNav.php [hunk 4]
```diff
@@ -346,13 +368,22 @@ public function sub_nav() {
...
+				esc_attr( $context ),
+				esc_attr( $context_id ),
+				$selected,
+				esc_attr( $type['name'] ),
+				esc_url( $profile_link . RTMEDIA_MEDIA_SLUG . '/' . constant( 'RTMEDIA_' . $name . '_SLUG' ) . '/' ),
+				$type_label,
+				esc_html( ( isset( $counts['total'][ $type['name'] ] ) ) ? rtmedia_number_to_human_readable( $counts['total'][ $type['name'] ] ) : 0 )
 			);
+
+			echo apply_filters( 'rtmedia_sub_nav_' . $type['name'], $nav_li, $type['name'] ); // @codingStandardsIgnoreLine
```
Comment (chandrapatel) [inline app/main/controllers/template/RTMediaNav.php:386]: This syntax of ignoring PHPCS is deprecated. Use `// phpcs:ignore <rule>`.

File: templates/upload/uploader.php [hunk 5]
```diff
@@ -130,3 +146,18 @@
 	</div>
 	<?php
 }
+
+/**
+ * Display hidden fields.
+ *
+ * @param string $key   Input key.
+ * @param string $value Input value.
+ *
```
Comment (chandrapatel) [inline templates/upload/uploader.php:155]: Add `@since` with upcoming new version release number for new functions.

**Review by chandrapatel — APPROVED** [review]
Approved with minor feedback.

File: app/main/controllers/template/RTMediaNav.php (hunk 2 above)
Comment (vishalkakadiya) [reply app/main/controllers/template/RTMediaNav.php]: Done

File: app/main/controllers/template/RTMediaNav.php (hunk 1 above)
Comment (vishalkakadiya) [reply app/main/controllers/template/RTMediaNav.php:290]: Done

**Pull request merged**